packages/tocco-theme/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "devDependencies": {
     "bootstrap-sass": "^3.3.7",
-    "react-select": "^1.0.0-rc.10"
+    "react-select": "^1.0.0-rc.10",
+    "tocco-ui": "^0.1.14"
```

Comment:
are you sure about this? I think this adds a circular dependency because ui should depend on theme and not the other way around?

Reply:
tocco-theme is not imported in tocco-ui. Hence I do not see any circular dependency.

tocco-ui is imported in tocco-theme to reuse utility function trimDecimalPlaces in modularScale.js. Probably it is better to remove the dependency an replace trimDecimalPlaces by Lodash [_.round](https://lodash.com/docs/4.17.10#round). What do you think?

Reply:
sound better yes